src/home/ActionsCarousel.test.tsx

```diff
@@ -9,18 +9,49 @@ import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import { CloseIcon } from 'src/navigator/types'
 import { mocked } from 'ts-jest/utils'
+import { MockStoreEnhanced } from 'redux-mock-store'
+import { createMockStore } from 'test/utils'
+import { Provider } from 'react-redux'
+
+function getStore(shouldShowSwapAction: boolean) {
+  return createMockStore({
+    app: {
+      showSwapMenuInDrawerMenu: shouldShowSwapAction,
+    },
+  })
+}
 
 describe('ActionsCarousel', () => {
+  let store: MockStoreEnhanced<{}>
+  beforeEach(() => {
+    store = getStore(true)
```

Comment:
nit: `getStore(true)` is small enough that this beforeEach seems like an overkill 😄